README.md

```diff
@@ -1,70 +1,119 @@
-# Getting Started with Create React App
+### The Commander's Dashboard (CC-Dashboard) provides a situational awareness update for all users
```

Comment:
Great attention to detail in this readme.md!  Just as with your API's readme, this is very thorough, well thought out, and gives the user all the information they need to get a good idea of what to expect.

Tips - It can be a good idea, especially with a client-side app, to show some of the UX elements, such as by including a few screenshots, to show the polished and operational behaviour of the app.  Check out some of the readmes on popular packages on github (usually reachable easily from a link via their npm page) to see what some of the pros have done.